common/src/lazy_pages.rs

```diff
@@ -101,20 +94,14 @@ pub fn protect_pages_and_init_info(
 
 /// Lazy pages contract post execution actions
 pub fn post_execution_actions(
-    memory_pages: &mut BTreeMap<PageNumber, Box<PageBuf>>,
+    memory_pages: &mut BTreeMap<PageNumber, PageBuf>,
     wasm_mem_begin_addr: u64,
 ) -> Result<(), Error> {
     // Loads data for released lazy pages. Data which was before execution.
     let released_pages = gear_ri::get_released_pages();
     for page in released_pages {
         let data = gear_ri::get_released_page_old_data(page)?;
-        let page_data = PageBuf::try_from(data).map_err(|_| Error::VecToPageData)?;
-        if memory_pages
```

Comment:
Why this check have been removed?

Reply:
Because now we receive PageBuf from gear_ri